hive/src/main/java/org/apache/iceberg/hive/HiveCatalog.java

```diff
@@ -188,6 +300,33 @@ protected String defaultWarehouseLocation(TableIdentifier tableIdentifier) {
         tableIdentifier.name());
   }
 
+  public  Map<String, String> getMetafrpmhiveDb(Database database) {
+    Map<String, String> nameMap = new HashMap<>();
+
+    nameMap.put("name", database.getName());
+    nameMap.put("location", database.getLocationUri());
+    nameMap.put("comment", database.getDescription());
+    database.getParameters().forEach(nameMap::put);
+
+    return nameMap;
+  }
+
+  public  Database nameSpaceToHiveDb(Namespace namespace) {
+    String warehouseLocation = conf.get("hive.metastore.warehouse.dir");
+    Preconditions.checkNotNull(
+        warehouseLocation,
+        "Warehouse location is not set: hive.metastore.warehouse.dir=null");
+    Joiner slash = Joiner.on("/");
```

Comment:
Why there is no check for empty namespace here?

Reply:
ok i will fix it